src/main/java/com/uber/cadence/internal/metrics/MetricsType.java

```diff
@@ -161,4 +161,11 @@
 
   public static final String NON_DETERMINISTIC_ERROR =
       CADENCE_METRICS_PREFIX + "non-deterministic-error";
+
+  public static final String SHADOWING_NON_DETERMINISTIC_ERROR =
+          CADENCE_METRICS_PREFIX + "shadowing-non-deterministic-error";
+  public static final String SHADOWING_SKIPPED_COUNT =
+          CADENCE_METRICS_PREFIX + "shadowing-skipped-workflow";
+  public static final String SHADOWING_SUCCESS_COUNT =
+          CADENCE_METRICS_PREFIX + "shadowing-success-workflow";
```

Comment:
shall we make these metrics the same as those defined in go-client?

Also looks like this part should belong to the replay activity diff?

Reply:
Moved to replay activity PR